subiquity/controllers/filesystem.py

```diff
@@ -419,13 +419,13 @@ def delete_raid(self, raid):
         self.model.remove_raid(raid)
 
     def create_volgroup(self, spec):
-        devices = set()
+        devices = []
```

Comment:
One can't add the same disk twice, so set() seems in order here.  What's the reason for the change?

Reply:
Well the discovery I made yesterday that the order matters to mdadm. But I agree this is conceptually bogus. Probably the better fix is to pass them around as a set and sort them (consistently but arbitrarily) in get_raid_sizes and when rendering the config.

That will let me make this change much smaller, which is nice...